Packs/Jira/Scripts/JiraAddComment/JiraAddComment.py

```diff
@@ -0,0 +1,24 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def add_comment(args: Dict[str, Any]) -> CommandResults:
+    comment_body = args.get('comment', '')
+    tags = argToList(args.get('tags', ''))
```

Comment:
The thing is this, we create the note in XSOAR with the relevant tag, and then, this note will get picked up as part of the mirror out process, where the code (that is in JiraV3.py) will check if the picked up note has the correct tag (the tag that is configured in the instance's configuration), and if it does, it will add the comment.
In other words, this script helps the user create a note with the tag in one hit, instead of first creating the note, and then tagging it. Makes sense?